### DISCUSSION TITLE
Show all roles under a user who has multiple roles

### DISCUSSION THREAD

**Discussion by lachie-92** [spatie/laravel-permission#1733] [Q&A] [answered]
Hi, does anyone know how to show multiple roles that are under a user? I have setup my permissions so a user can be under multiple roles, which is set easily enough using the following

        $user->assignRole(['Root', 'IT', 'HR']); // assigning roles

I am however having trouble showing all roles a user relongs to in the same way? 
Has anyone done this and knows how to? I simply want to show on a page which role a user belongs to.

**Comment by drbyte** [comment, marked as answer]
https://spatie.be/docs/laravel-permission/v4/basic-usage/basic-usage

```php
// get the names of the user's roles
$roles = $user->getRoleNames(); // Returns a collection
```

**Reply by smongare** [reply]
In my case, this is bring an error: Method Illuminate\Database\Eloquent\Collection::getRoleNames does not exist.
